### PR TITLE
Adding Contributors to README.md (Front-end)

### LINKED ISSUE #88: Update the Readme with Contributirs
### Description
Update the contributor's section of Readme.
Requirement: Every contributor's should be there and clicking on it should open his Github profile.

### Screenshots/testimonials

Screenshot from 2020-09-25 15-59-58

### PR DESCRIPTION
**Pull request by hariom1625**
fixes #88 
<Do not delete this template>
Description: <provide a description> Added Contributors list in the README.md .
In the picture attached below, there are two types of layout present, one with an emoticon below the name and one without, I need your views which one looks better. The emoticons can be varied.

#### Testimonial
contributors

### CODE REVIEW

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
One last this is you can squash these commit and then push again! 
And you're good to go 🎉

**Comment by tusharjain0022** [comment]
I think without the emoticons it would look nice, and we don't have to bother much about which emoticons to be used with whom otherwise both looks cool to me! 😄.

**Review by tusharjain0022 — APPROVED** [review]

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  "contributors": [
+    {
+      "login": "ankiiitraj",
+      "name": "Ankit Raj",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/48787278?v=4",
+      "profile": "https://github.com/ankiiitraj",
+      "contributions": [
+        "code"
+      ]
+<<<<<<< HEAD
```
Comment (anktrj) [inline .all-contributorsrc]: whoops . . . forgot these?

Comment (hariom1625) [reply .all-contributorsrc]: Don't know-how, it isn't visible in my code editor, showing only on GitHub.😕
Will fix !

**Review by ankiiitraj — APPROVED** [review]

**Comment by hariom1625** [comment]
> One last this is you can squash these commit and then push again!
> And you're good to go 🎉

Done and Thank you for suggesting this :-)

**Comment by hariom1625** [comment]
> I think without the emoticons it would look nice, and we don't have to bother much about which emoticons to be used with whom otherwise both looks cool to me! 😄.

Right 💯

**Pull request merged**